charts/prometheus-tplink-plug-exporter/values.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: tplink-plug-exporter
+
+image:
+  name: bboerst/tplink-plug-exporter
```

Comment:
This image was built and pushed from here: https://github.com/bboerst/tplink-plug-exporter/blob/version-bumps/Dockerfile which includes some version bumps.